default/scripting/macros/misc.py

```diff
@@ -60,3 +68,23 @@ def DESCRIPTION_EFFECTSGROUP_MACRO(desc: str):
         1 + 0.5 * StatisticIf(float, condition=IsTarget & EmpireHasAdoptedPolicy(name="PLC_POPULATION"))
     )  # +50% growth with population policy
 )
+
+
+# @1@ Species key
```

Comment:
You don't need this line anymore.

Reply:
What about leaving it as a remark, a comment?

Reply:
Why do you need it?  It just means the first argument of the helper, and you count the Python function's arguments to find it.